scimono-client/src/main/java/com/sap/scimono/client/authentication/OauthClientCredentialsAuthenticator.java

```diff
@@ -37,7 +38,12 @@ public String authenticate() {
       tokenProperties = retrieveOauthToken();
     }
 
-    return tokenProperties.getTokenType() + " " + tokenProperties.getAccessToken();
+    String tokenType = tokenProperties.getTokenType();
+    if ((tokenType == null) || "Bearer".equalsIgnoreCase(tokenType)) {
```

Comment:
null check is redundant here. It is satisfied by the second check

Reply:
Fixed